### PR TITLE
Some cleanups for the generate command.

### PR DESCRIPTION
**Pull request by Kashomon**

### CODE REVIEW

File: pkg/commands/generate/get_command.go [hunk 1]
```diff
@@ -18,25 +18,24 @@ package generate
...
 func GetCommand() *cobra.Command {
 	var filepath string
 	var componentName string
+
 	cmd := &cobra.Command{
 		Use:   "generate",
 		Short: "Generate placeholder components",
 		Long:  "Generate placeholder components",
 		Run: func(cmd *cobra.Command, _ []string) {
-			if filepath == "" {
```
Comment (BradfordMedeiros) [inline pkg/commands/generate/get_command.go]: Shouldn't we check this?

File: pkg/generate/generate.go [hunk 2]
```diff
@@ -15,15 +15,19 @@
...
+	"os"
+	"path"
+	"text/template"
+
+	log "k8s.io/klog"
+)
 
 const componentBuilder = `apiVersion: bundle.gke.io/v1alpha1
 kind: ComponentBuilder
-componentName: etcd-component
```
Comment (BradfordMedeiros) [inline pkg/generate/generate.go]: woopsie good catch.

File: pkg/generate/generate.go [hunk 3]
```diff
@@ -102,24 +108,26 @@ func Create(filepath string, name string) error {
...
+	var compText bytes.Buffer
+	compTmpl.Execute(&compText, replacement)
+
+	if err := ioutil.WriteFile(path.Join(dir, replacement.Name+"-builder.yaml"), compText.Bytes(), 0666); err != nil {
+		log.Exit(err)
 	}
-	if err := ioutil.WriteFile(path.Join(filepath, "sample-deployment.yaml"), deploymentText.Bytes(), 0666); err != nil {
-		writeErr = err
+	if err := ioutil.WriteFile(path.Join(dir, "sample-deployment.yaml"), deploymentText.Bytes(), 0666); err != nil {
+		log.Exit(err)
```
Comment (BradfordMedeiros) [inline pkg/generate/generate.go]: with log.exit you're making assumptions about how this is used.  Ideally you should allow the calling context retain the ability to decide how to handle the error.

File: pkg/commands/generate/get_command.go (hunk 1 above)
Comment (Kashomon) [reply pkg/commands/generate/get_command.go]: I changed it to rely on the CWD by default (by essentially allowing an empty filepath).

File: pkg/generate/generate.go (hunk 3 above)
Comment (Kashomon) [reply pkg/generate/generate.go]: Good point. I'll just return the error.

Comment (Kashomon) [reply pkg/generate/generate.go]: I'll do that in a follow-up. I've got some more cleanups I was thinking about.

**Pull request merged**